src/api/client.ts

```diff
@@ -86,6 +87,7 @@ function requestHandler(config: AxiosRequestConfig): AxiosRequestConfig {
   }
 
   config.headers[AppVersionHeaderName] = APP_VERSION;
+  config.headers[PlatformVersionHeaderName] = Platform.Version;
```

Comment:
This looks fine for adding the platform version, but why isn't platform added in this function (it's also missing from `src/api/headers.ts`). Same with app id.

Reply:
I agree. We should move adding of the 'atb-app-identifier' and 'atb-app-platform' headers to here as well, instead of in `requestIdTokenHandler`.